### ISSUE TITLE
Release v0.3

### ISSUE THREAD

**Issue by blegat** [JuliaAlgebra/StarAlgebras.jl#63]
Symmetry reduction now work on SumOfSquares master with StarAlgebras the same way it used to in the previous version so I can do a new release once StarAlgebras is released.
Any blocker to release ? We could wait for https://github.com/JuliaAlgebra/StarAlgebras.jl/pull/62

**Comment by kalmarek** [comment]
yeah, I need your input on #62. If that's enough for you I'm ready to merge and release ;)

**Comment by kalmarek** [comment]
Roadmap:
- [ ] Remove MTables for now (can be re-added in minor version)
- [ ] Agree on quadratic forms (see #62 #64)
- [ ] Adjust master of SymbolicWedderburn

**Comment by kalmarek** [comment]
@blegat I think it is unsustainable to keep it so long in the loop; I think this is ready for version 0.3

maybe then you could work on this and release SumOfSquares with v0.4 ?

**Comment by blegat** [comment]
Yes, we can release, maybe we won't need anything breaking so v0.3.1 would be fine

**Comment by kalmarek** [comment]
versions come dozen a penny, even 0.61.37 would be fine;
